demo/src/plugins/dashboard/screens/dashboard/index.jsx

```diff
@@ -3,37 +3,74 @@ import PropTypes from 'prop-types';
 import NodeStatus from '@gdbots/schemas/gdbots/ncr/enums/NodeStatus';
 import { Col, Container, Row, Screen } from '@triniti/admin-ui-plugin/components';
 import TopArticles from '../../components/top-articles';
+import ActiveEdits from '../../components/active-edits';
+
+const Dashboard = ({ location, match: { params: { tab } } }) => {
+  const tabs = [
+    {
+      to: 'news',
+      text: 'news',
+    },
+    {
+      to: 'active',
+      text: 'active',
+    },
+  ];
+
+
+  return (
+    <Screen
+      tabs={tabs}
+      maxWidth="100%"
+      title="Dashboard"
+      breadcrumbs={{ length: null }}
```

Comment:
@nanetteCodes add a comment here that this is a temp workaround until admin-ui gets a better solution to not having breadcrumbs

Reply:
updated